src/Hyperlink/Hyperlink.test.jsx

```diff
@@ -38,7 +38,7 @@ describe('correct rendering', () => {
   it('renders external Hyperlink', () => {
     const wrapper = mount(<Hyperlink {...externalLinkProps} />);
 
-    expect(wrapper.find('span')).toHaveLength(2);
+    expect(wrapper.find('span')).toHaveLength(3);
```

Comment:
There are now 3 spans, since `Icon` now renders its own `span` for the `screenReaderText`.